### PR TITLE
[FLINK-34180] Migrate doc website from ververica to flink

### PR DESCRIPTION
**Pull request by GOODBOY008**
[FLINK-34180] Migrate doc website from ververica to flink.

### CODE REVIEW

File: .dlc.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (PatrickRen) [inline .dlc.json:1]: What's the purpose of this file?

**Review by PatrickRen — COMMENTED** [review]
@GOODBOY008 Thanks for the awesome work! I left one comment.

File: .dlc.json (hunk 1 above)
Comment (GOODBOY008) [reply .dlc.json:1]: This file I would like to check dead link in doc

Comment (PatrickRen) [reply .dlc.json:1]: What about we introduce this file after we enable the dead link check in the future? I'm not sure links in this file currently are related to Flink CDC's doc.

**Review by PatrickRen — APPROVED** [review]
@GOODBOY008 Thanks for the PR! LGTM overall. Let's merge first and see if the doc can be built successfully.

**Pull request merged**